### ISSUE TITLE
Question about performance

### ISSUE THREAD

**Issue by dzmitry-zimin** [jitsi-contrib/jitsi-helm#47]
Hey, guys does anybody tested jitsi's performance? We've deployed jitsi helm chart in kubernetes (on docker-desktop). Then we did a performance test with in about 40 participants, we use one jvb. We noticed 3 things: 
1. Some screens were black;
2. After adding in about more than 35 participants some screens became hangs(video, audio).
3. Then we are all together did page refresh to check a how fast we can join again to room, eventually we've got a pause in about 1-1.5 min.

Our resources 2 CPU, 4 RAM, Network 100 Mbps (max).

My question: what we have to fix to avoid these problems?

**Comment by spijet** [comment]
Hello @dzmitry-zimin!

Here's a couple of things you should consider checking:
1. Make sure that you have UDP transport set up and working properly (there are no timeout messages in JVB logs);
2. Check if WebSockets are working (using `jitsi-web` logs and Development Console in browser) — without those all streams would be limited to sound and lowest quality video;
3. Check if the clients you use have agreed on the video and audio codec. I've had some issues with Firefox where it either wouldn't receive VP9 video from other users or force everyone to downgrade to VP8. In my experience, we got best results on [Jitsi Meet app](https://github.com/jitsi/jitsi-meet-electron/).

Also, do keep in mind that Jitsi Meet uses Unicast to distribute audio and video streams among users. 
To put it simply, with `N` users in a room JVB will be receiving N audio and up to `3*N` video streams (depending on the video quality each user requests for every other user's video feed), and then send all these streams back to every single user.  A high-quality video stream takes up to 4 megabits of bandwidth, so it won't take many users in a single room to oversaturate your 100Mbps uplink.
